libs/use-multi-media-streams/src/types.ts

```diff
@@ -44,6 +58,8 @@ export interface Resolution {
 }
 
 export interface Stream {
+  bitrate: number;
```

Comment:
As I mentioned above, bitrate and codec are purely for publishers, this hook should be more generic. We should map and manage these streams and the bitrate/codec in the publisher hook.